Purchasing.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31702.278
+# Visual Studio Version 17
```

Comment:
did you deliberately upgrade this or did vs do it on its own?

Reply:
did it on its own